frontendActions/src/main/resources/actions/ResourceBundle.properties

```diff
@@ -0,0 +1,33 @@
+
+# Context menu
```

Comment:
This is more wieldy since it can be easily accessed from Java code (as opposed to when it was dug deep in plugin.xml) + it can be i18ned (which I don't think we'll ever do anyway xD)

Reply:
Y not? :poland: :rofl: 
> Rebazuj obecną gałąź na nadrzędną

Reply:
awwww :star_struck: